internal/pkg/scaffold/cmd.go

```diff
@@ -160,7 +160,7 @@ func main() {
 	}
 	// Create Service object to expose the metrics port(s).
 	service, err := metrics.CreateMetricsService(ctx, cfg, servicePorts)
-	if err != nil {
+	if err != nil && err != k8sutil.ErrRunLocal {
```

Comment:
This error will never be `ErrRunLocal`. We check for that error, log that we're skipping, and then always return nil:
https://github.com/operator-framework/operator-sdk/blob/98ab00584b98f2d61e5d8505a4e3173ee37d2de3/pkg/metrics/metrics.go#L54-L60

Reply:
We are printing out an error there. I found the problem, but will handle that in another PR